internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -735,7 +735,7 @@ func getPullRequestInfoFromPayload(
 
 	return &pb.PullRequest{
 		Url:      prUrl,
-		Number:   int32(prNumber),
+		Number:   int64(prNumber),
```

Comment:
I'm assuming we can't use `JQReadFrom[int64]` on line 721 because the underlying API understands that JSON numbers are floats?